### PR TITLE
hacker news site added

### PR DESCRIPTION
**Pull request by Stroller15**
## Changes proposed

hacker news site added

solved issue #455

### CODE REVIEW

**Comment by Stroller15** [comment]
@TreciaKS pls review my PR

**Review by TreciaKS — COMMENTED** [review]
@Stroller15 it seems you modified over 11 files just to add one site URL to the PR. Why is that? I believe this is the reason your PR is resulting in this error:https://github.com/orgs/community/discussions/25706. Otherwise the Hacker News object added looks great to me. You'll have to wait for the approval of your PR.

**Comment by TreciaKS** [comment]
@CBID2 May you suggest something?

File: database/frontend/design-inspirations.json
Comment (CBID2) [inline database/frontend/design-inspirations.json:144]:
```suggestion
        "description": "The smarter way to research user interface flows, screen, email best practices, and getting unstuck is by learning from proven products.",
        "url": "https://pageflows.com/",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Stroller15! 😄 The Hacker News addition looks pretty good. In regards to the other sources, I just added a suggestion to improve one of the descriptions.

**Review by TreciaKS — COMMENTED** [review]
@Stroller15 I just realised that the changes you've made here you've also made in this [PR](https://github.com/rupali-codes/LinksHub/pull/467) for issue #467 

I feel this would cause conflcts. @CBID2 What do you think about this? (Forgive me for being on a pinging spree this evening for you 😓 )

**Comment by CBID2** [comment]
> @Stroller15 I just realised that the changes you've made here you've also made in this [PR](https://github.com/rupali-codes/LinksHub/pull/467) for issue #467
> 
> I feel this would cause conflcts. @CBID2 What do you think about this? (Forgive me for being on a pinging spree this evening for you 😓 )

I don'

> @Stroller15 I just realised that the changes you've made here you've also made in this [PR](https://github.com/rupali-codes/LinksHub/pull/467) for issue #467
> 
> I feel this would cause conflcts. @CBID2 What do you think about this? (Forgive me for being on a pinging spree this evening for you 😓 )

According to [this](https://github.com/github/codeql/issues/8843), this seems to be more associated with the GitHub Actions.

**Comment by TreciaKS** [comment]
Ohh I see, thanks for the link. I read through it.

I've learned something 👍

**Comment by CBID2** [comment]
> Ohh I see, thanks for the link. I read through it.
> 
> I've learned something 👍

Great! 👍🏾 Keep me and @rupali-codes posted! 😄

**Review by rupali-codes — COMMENTED** [review]
Please add the ONLY files you've made changes in :)

**Comment by rupali-codes** [comment]
Hey, could you open a fresh PR containing the only changes you have made directly to MAIN branch?

**Comment by Stroller15** [comment]
>Hey, could you open a fresh PR containing the only changes you have made directly to MAIN branch?
hey @rupali-codes I made changes in dev branch as @TreciaKS said earlier

**Pull request closed without merging**
